### DISCUSSION TITLE
Custom Fonts and Tilde Characters

### DISCUSSION THREAD

**Discussion by SkydivinGirl** [mrothenbuecher/zpl-rest#18] [Q&A]
This project is great, and I'm finding good use for it! I am wondering if there's a way to display a downloaded font in the preview window.  It's not in the ZPL, but stored in the FLASH of our devices.  Having a way to simulate that in the preview would be a 'nice to have'.  It doesn't affect the functionality at all.

One issue I'm having is using the tilde symbol as text for the parameters.  When printing, the field shows everything up until it reaches the first tilde.  The data returned from the POST shows that the tilde is being sent in the ZPL as %7e.

Thanks for a great program!

Heather

**Comment by mrothenbuecher** [comment]
Hi,

Sounds a bit like this issue https://github.com/mrothenbuecher/zpl-rest/issues/1
